### ISSUE TITLE
[feature] Full 64-bit os for Rpi Zero2W?

### ISSUE THREAD

**Issue by amenekowo** [aluminum-ice/pwnagotchi#54]
**Is your feature request related to a problem? Please describe.**  
We just dropped the support of Rpi Zero W in 1.7.7, so I wondered that we could use a arm64 image of raspbian at build. According to Jeff's YouTube video, arm64 seems have a better performance, so maybe it helps pwnagotchi too.  

**Describe the solution you'd like**
Make built OS fully arm64  

**Describe alternatives you've considered**

**Additional context**
I think it could be possible because we build nexmon from source. Also, tensorflow could also build for arm64. Just wondered how much time we need to build this.

**Comment by aluminum-ice** [comment]
You're welcome to try it and submit PRs. At this time, my RPiZ2W works perfectly so I'm not spending time adding more features like this. It's not needed.

**Comment by waterfallm** [comment]
i might have something soon for this as i'm looking at bumping the version of the OS to get my GeekWorm x306 working

**Comment by aluminum-ice** [comment]
@waterfallm Did you ever test your idea?

**Comment by Opalinus** [comment]
@jayofelony started a fork for 64-bit Raspberry Pi's. You can find it [here](https://github.com/jayofelony/pwnagotchi-bookworm)

**Comment by jayofelony** [comment]
> @jayofelony started a fork for 64-bit Raspberry Pi's. You can find it [here](https://github.com/jayofelony/pwnagotchi-bookworm)

🎉🎉 And boy oh boy does it work well.. 🫡

**Comment by aluminum-ice** [comment]
> @jayofelony started a fork for 64-bit Raspberry Pi's. You can find it [here](https://github.com/jayofelony/pwnagotchi-bookworm)

Yes, it's good work. This is my hobby tho and I like working on my fork. 

I'm close to a fully working 64bit stable baseline3 version. Been running it locally with a few remaining small fixes needed.

**Comment by Opalinus** [comment]
> > @jayofelony started a fork for 64-bit Raspberry Pi's. You can find it [here](https://github.com/jayofelony/pwnagotchi-bookworm)
> 
> Yes, it's good work. This is my hobby tho and I like working on my fork.
> 
> I'm close to a fully working 64bit stable baseline3 version. Been running it locally with a few remaining small fixes needed.

Can't wait to test it, since my PiSugar2 doesn't seem to work with armv7l. I found this [issue](https://github.com/PiSugar/pisugar-power-manager-rs/issues/53)

**Comment by Maynardnaze** [comment]
Are we waiting for Waveshare to release an update?
